src/Components/ImagesTable/ImagesTable.js

```diff
@@ -175,7 +205,20 @@ const ImagesTable = () => {
             Create image
           </Link>
           <EmptyStateSecondaryActions>
-            <DocumentationButton />
+            <Button
+              component="a"
+              target="_blank"
+              variant="link"
+              icon={<ExternalLinkAltIcon />}
+              iconPosition="right"
+              isInline
+              href={
+                'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/creating_customized_rhel_images_using_the_image_builder_service/index'
+              }
+              className="pf-u-pt-md"
+            >
+              Image builder for RPM - DNF documentation
```

Comment:
This has spaces but on line 177 it is without spaces. I think without spaces looks better. According to some random grammar websites, in US English an en dash should not have spaces around it. We can remove the spaces from the tabs to be consistent in another PR.

Reply:
Agree, spaces removed! I've tried to remove them everywhere, but missed this